docroot/sites/facilities.uiowa.edu/modules/facilities_core/sass/facilities_core.scss

```
@@ -0,0 +1 @@
+// We need a blank file for `yarn workspace facilities_core build` to execute properly.
```

Is this just because package.json was a copy/pasta and there are SASS related actions in there? If it is easier to copy paste for build consistency that is fine with me. If it is just as easy to slim down the footprint, I am for that as well.

```suggestion
// We need a blank file for the generic build process to execute properly.
```

---

When you run the yarn workspace command, it will only execute if that file is there. The js compilation happens after the SCSS compilation, and if the SCSS compilation doesnt find source files, it stops, thus never getting to the JS compilation

---

I hope that makes sense?

---

Right, which is the way package.json was previously written/copied and not specific to the needs of this. The file could be adjusted to remove the CSS/SASS portions and just to build JS.

---

Totally fair. I can change this.

---

I have done the changing!